# Flight in Flux

Neva Morgan (Colorado State University)  
Mia Colangelo (Colorado State University)

# **Flight in Flux:** Analyzing American Bittern (Botaurus lentiginosus) Migration Timing and Climate Change

### **Introduction:**

Climate change poses many questions relating to systems across the globe, especially ecological and evolutionary systems. One system which could potentially be heavily impacted is migration. Bird migration timing is critical for ensuring species survival and ecological balance. As anthropogenic climate change has altered seasonal patterns, there is a potential disruption to migration timing, impacting birds’ breeding success, food availability, and overall population health. Wetland birds are expected to be affected negatively by a considerable margin with the presence of a changing climate altering their regular migratory ranges and patterns (Steen et al., 2012). The American Bittern (Botaurus lentiginosus), for example, depends on the thriving vegetation of marshlands for camouflage and nesting, as they are known to nest on the ground.

Bird migration range and timing is critical for ensuring species survival and ecological balance. However, climate change has altered seasonal patterns, potentially disrupting migration timing and impacting breeding success, food availability, and overall population health. Understanding these shifts can inform conservation strategies and mitigate risks to migratory bird species. Additionally, information gained from analyzing one species can be used to make inferences about the potential shifts in other similar species.

### **Abstract:**

The American Bittern is a solitary, cryptic species that relies on wetlands for breeding and overwintering. Typically, these birds breed in freshwater marshes across Canada and the northern United States before migrating south to the southeastern U.S., Mexico, and Central America for the winter. However, increasing global temperatures, shifts in precipitation, and habitat loss due to climate change threaten to disrupt these migration patterns. These changes are affecting not only local species, but also have the potential to shift patterns thousands of miles away (Smith 2022)  
Past studies on migratory birds indicate that climate change is leading to earlier spring arrivals, delayed fall departures, and shifts in overall migration routes (La Sorte & Thompson, 2007). Additional research has shown that for some birds, more severe dry seasons can decrease survival rates for the following migration (Georgetown 2023). For wetland-dependent birds like the American Bittern, changes in water levels due to altered precipitation cycles could further complicate their ability to find suitable stopover sites and breeding grounds. Given that wetlands are already among the most threatened ecosystems in North America, with significant losses due to urbanization and agriculture, climate-induced changes pose an additional challenge (Dahl, 2011). Changes such as rising water levels impact marshlands and wetlands especially, and while the birds inhabiting them are not typically listed as endangered, these changes put them at risk for habitat loss (Gieger 2023) Studying the migration patterns of the American Bittern under a changing climate is essential for predicting future population trends and informing conservation strategies. By understanding how climate variables influence their movements, conservationists can prioritize habitat protection, restoration, and adaptive management to mitigate potential negative impacts.

### **Data:**

#### Climate Data Online (NOAA Global Summary of the Month - U.S. Specific):

With NOAA’s Climate Data Online search, we focused on nine weather stations located between Arizona (two weather stations: Phoenix and Kingsman), California (three weather stations: Lake Tahoe, San Diego, and Santa Rosa), Oregon (two weather stations: Chiloquin and Hermiston), and Washington (two weather stations Shelton and Spokane), to map multiple points of migratory status for the American Bittern. The files were requested through eBird, and then downloaded into a readable CSV for Excel to initially portion the data into the portions necessary for our analysis. Each weather station collected data that contained the specific information for the weather station, temperature, precipitation, and wind-related metrics (of which are negligible for our testing metrics).

Variables within the Datasets contain:

-   Metadata: Station name (NAME), location (LATITUDE, LONGITUDE, ELEVATION), and observation date (DATE)

-   Climate Variables: Includes average and extreme temperatures (e.g., ADPT, ASLP, AWBT), precipitation statistics (e.g., DP01, DP10), and wind metrics (AWND)

-   Attributes and Flags: Columns such as \*\_ATTRIBUTES and logical flags provide metadata on data quality and source

-   Format: Character (station info and attribute flags), numeric (climate measures), and logical (indicator variables)

#### American Bittern - eBird Data:

This dataset was sourced from eBird and includes observations of American Bitterns (Botaurus lentiginosus) reported by citizen scientists. The dataset spans from 2000 to 2024 and includes over 70,000 observations.

-   Geographic Coverage: Arizona, California, Oregon, and Washington within the U.S.

-   Metadata: Observation Date Time Observations Started Latitude Longitude

-   Bird Details: Observation Count Breeding Code Behavior Code

-   Format: Primarily character and logical columns, with a few numeric fields (e.g., coordinates, observer count)

### **Methods**

To examine the effects climate change will have on the migratory timing of the American Bittern, we used a monthly summary of climate data from nine recorded NOAA weather stations within Arizona, California, Oregon, and Washington and observational data of their abundance within those states from eBird.

#### Data Acquiring:

Climate data within nine weather stations stretched between Arizona, California, Oregon, and Washington by NOAA Global Monthly Summaries, were recorded on a scale between 1947-2025. By using ‘R’ resources, we cleaned this data, removing unnecessary attributes for our study, converting units to be consistent throughout each dataset, and removing unnecessary data from past years to focus on data between 2000-2024.

#### Data Processing / Seasonal Aggregation:

Each weather station recorded the seasonal averages of temperature and precipitation, which were then manipulated into a cleaned version of that data, removing unnecessary wind records. With that, we created a list item, using ‘RStudio’, that combined the contents of the dataframes, to join them all together, ensuring columns and the contents of the dataframes are not duplicated or N/Aed. We then integrated a seasonal component into the data denoted by month: Winter = 12 (Dec.), 1 (Jan.), 2 (Feb.); Spring = 3 (Mar.), 4 (Apr.), 5 (May); Summer = 6 (Jun.), 7 (Jul.), 8 (Aug.); and Fall = 9 (Sep.), 10 (Oct.), 11 (Nov.). The seasonal component was also applied to the American Bittern migration dataframe, which was later converted to a tibble for later analysis.

#### Migratory Timing Analysis of American Bittern:

Observational data was provided to understand the timing of Bitterns within our four selected states, by grouping month and year to understand temporal patterns of their appearance based on the seasonal presence. Using a time series visualization to detect changes within the migratory patterns based on their seasonality between 2000 and 2024.

#### Comparing Seasonal Trends of Climate Variability and Bittern Observation:

To understand the correlation between seasonal changes for all the weather stations, we averaged the temperature and precipitation of each month per year. We then evaluated the long-term changes in bittern observation from plot layouts used before, by visually plotting the changes in the presence of climate seasonality.

To evaluate the influence of climate variability on American Bittern observations from 2000 to 2024, we fit a linear mixed effects model using the lmer function from the lme4 package in R. The response variable was the count of observations per year and state. Fixed effects included average temperature (avg_temp) and average precipitation (avg_precp), representing climatic conditions. To account for non-independence among observations across years and states, we included season year and state indicators (AZ, CA, OR, and WA) as random intercepts. This approach allowed us to control for unobserved heterogeneity due to temporal and spatial variation, isolating the effects of climate variables on bird observations. All data and observations were analyzed using RStudio version 2025.05.0 Build 496.

### **Results:**

##### Coding and Figures:

In [ ]:
# Beginning Mumbo Jumbo Libraries:

library(flextable) 

── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.4     ✔ purrr     1.0.4
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ ggplot2   3.5.2     ✔ tibble    3.2.1
✔ lubridate 1.9.4     ✔ tidyr     1.3.1
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ purrr::compose() masks flextable::compose()
✖ dplyr::filter()  masks stats::filter()
✖ dplyr::lag()     masks stats::lag()
ℹ Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors

── Attaching packages ────────────────────────────────────── tidymodels 1.3.0 ──
✔ broom        1.0.8     ✔ rsample      1.3.0
✔ dials        1.4.0     ✔ tune         1.3.0
✔ infer        1.0.8     ✔ workflows    1.2.0
✔ modeldata    1.4.0     ✔ workflowsets 1.1.0
✔ parsnip      1.3.1     ✔ yardstick    1.3.2
✔ recipes      1.3.0     

── Conflicts ───────────────────────────────────────── tidymodels_conflicts() ──
✖ purrr::compose()  masks flextable::compose()
✖ scales::discard() masks purrr::discard()
✖ dplyr::filter()   masks stats::filter()
✖ recipes::fixed()  masks stringr::fixed()
✖ dplyr::lag()      masks stats::lag()
✖ yardstick::spec() masks readr::spec()
✖ recipes::step()   masks stats::step()


Attaching package: 'gridExtra'

The following object is masked from 'package:dplyr':

    combine


Attaching package: 'cowplot'

The following object is masked from 'package:lubridate':

    stamp

1.  Create time period for seasons

In [ ]:
# Read in all climate data
AZ_kingman <- read_csv("data/Climate Data/AZ-kingman-climate.csv")

Rows: 345 Columns: 90
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (42): STATION, DATE, NAME, ADPT_ATTRIBUTES, ASLP_ATTRIBUTES, ASTP_ATTRIB...
dbl (45): LATITUDE, LONGITUDE, ELEVATION, ADPT, ASLP, ASTP, AWBT, AWND, CDSD...
lgl  (3): DYFG_ATTRIBUTES, DYHF_ATTRIBUTES, DYTS_ATTRIBUTES

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

Rows: 319 Columns: 82
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (38): STATION, DATE, NAME, AWND_ATTRIBUTES, CDSD_ATTRIBUTES, CLDD_ATTRIB...
dbl (41): LATITUDE, LONGITUDE, ELEVATION, AWND, CDSD, CLDD, DP01, DP10, DP1X...
lgl  (3): DYFG_ATTRIBUTES, DYHF_ATTRIBUTES, DYTS_ATTRIBUTES

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

Rows: 657 Columns: 96
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (45): STATION, DATE, NAME, ADPT_ATTRIBUTES, ASLP_ATTRIBUTES, ASTP_ATTRIB...
dbl (48): LATITUDE, LONGITUDE, ELEVATION, ADPT, ASLP, ASTP, AWBT, AWND, CDSD...
lgl  (3): DYFG_ATTRIBUTES, DYHF_ATTRIBUTES, DYTS_ATTRIBUTES

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

Rows: 849 Columns: 84
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (39): STATION, DATE, NAME, AWND_ATTRIBUTES, CDSD_ATTRIBUTES, CLDD_ATTRIB...
dbl (42): LATITUDE, LONGITUDE, ELEVATION, AWND, CDSD, CLDD, DP01, DP10, DP1X...
lgl  (3): DYFG_ATTRIBUTES, DYHF_ATTRIBUTES, DYTS_ATTRIBUTES

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

Rows: 322 Columns: 96
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (45): STATION, DATE, NAME, ADPT_ATTRIBUTES, ASLP_ATTRIBUTES, ASTP_ATTRIB...
dbl (48): LATITUDE, LONGITUDE, ELEVATION, ADPT, ASLP, ASTP, AWBT, AWND, CDSD...
lgl  (3): DYFG_ATTRIBUTES, DYHF_ATTRIBUTES, DYTS_ATTRIBUTES

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

Rows: 511 Columns: 68
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (30): STATION, DATE, NAME, CLDD_ATTRIBUTES, DP01_ATTRIBUTES, DP10_ATTRIB...
dbl (36): LATITUDE, LONGITUDE, ELEVATION, CDSD, CDSD_ATTRIBUTES, CLDD, DP01,...
lgl  (2): DYFG_ATTRIBUTES, DYTS_ATTRIBUTES

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

Rows: 324 Columns: 82
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (38): STATION, DATE, NAME, AWND_ATTRIBUTES, CDSD_ATTRIBUTES, CLDD_ATTRIB...
dbl (41): LATITUDE, LONGITUDE, ELEVATION, AWND, CDSD, CLDD, DP01, DP10, DP1X...
lgl  (3): DYFG_ATTRIBUTES, DYHF_ATTRIBUTES, DYTS_ATTRIBUTES

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

Rows: 321 Columns: 96
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (45): STATION, DATE, NAME, ADPT_ATTRIBUTES, ASLP_ATTRIBUTES, ASTP_ATTRIB...
dbl (48): LATITUDE, LONGITUDE, ELEVATION, ADPT, ASLP, ASTP, AWBT, AWND, CDSD...
lgl  (3): DYFG_ATTRIBUTES, DYHF_ATTRIBUTES, DYTS_ATTRIBUTES

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

Rows: 344 Columns: 88
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (41): STATION, DATE, NAME, CLDD_ATTRIBUTES, DP01_ATTRIBUTES, DP10_ATTRIB...
dbl (45): LATITUDE, LONGITUDE, ELEVATION, CDSD, CDSD_ATTRIBUTES, CLDD, DP01,...
lgl  (2): DYFG_ATTRIBUTES, DYTS_ATTRIBUTES

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

1.  Building Seasons:

In [ ]:
# Changing into month and year
climate_data <- climate_data %>%
  mutate(
    DATE = paste0(DATE, "-01"), # Adding a day component since it was missing!
    DATE = as.Date(DATE, format = "%Y-%m-%d"),
    year = year(DATE),
    month = month(DATE)
  )


# Creating seasons:
climate_data <- climate_data %>%
  mutate(
    season = case_when(
      month %in% c(12, 1, 2)  ~ "Winter",
      month %in% c(3, 4, 5)   ~ "Spring",
      month %in% c(6, 7, 8)   ~ "Summer",
      month %in% c(9, 10, 11) ~ "Fall"
    ))


# Creating seasons with the year to compare for climate
climate_data <- climate_data %>%
  mutate(
    season_year = if_else(month == 12, year + 1, year)
  )


#Separating temp and precp to clean further!
avg_temp <- climate_data %>%
  mutate(TAVG_F = (TAVG * 9/5) + 32) %>%
  group_by(STATION, NAME, month, season, season_year) %>%
  summarize(avg_temp = mean(TAVG_F, na.rm = TRUE), .groups = "drop") %>%
  filter(season_year >= 2000, season_year <= 2024)

avg_prec <- climate_data %>%
  group_by(STATION, NAME, month, season, season_year)%>%
  summarize(avg_prec = mean(PRCP, na.rm = TRUE), .groups = "drop") %>%
  filter(season_year >= 2000, season_year <= 2024)

1.  Understanding the changes of average temperature and precipitation in a U.S. map!

<!-- -->

1.  Temperature Difference between 2000 - 2024 based on seasonality

The seasonal average temperature varies respectively within each recorded season showing similarities between Fall and Winter temperatures and Spring and Summer temperatures for all nine weather stations (Fig. 1).

In [ ]:
#TEMPERATURE!!!

#creating a separte coordination dataframe
station_coords <- climate_data %>%
  select(STATION, LATITUDE, LONGITUDE) %>%
  distinct()

#joining the two to relay to the map
avg_temp_map <- avg_temp %>%
  left_join(station_coords, by = "STATION")

#loading the u.s. map raster file!
us_map <- map_data("state")

#Creating map of avg temperature for Selected stations based on seasonality!

ggplot(avg_temp_map %>% filter(season_year >= 2000 & season_year <= 2024)) +
  geom_polygon(data = us_map, aes(x = long, y = lat, group = group),
               fill = "gray90", color = "white") +
  geom_point(aes(x = LONGITUDE, y = LATITUDE, color = avg_temp), size = 2) +
  scale_color_viridis_c(option = "plasma", name = "Avg Temp (°F)") +
  coord_fixed(1.3) +
  facet_wrap(~season) +
  labs(title = "Seasonal Average Temperatures by Station (2000–2024)",
       x = "", y = "") +
  theme_minimal()

All seasons but Spring showed several degree increase in average temperature from 2000 to 2024 (Fig. 2).

In [ ]:
# Let's compare first just what 2000 vs 2024 looks like!

# separating 2000 and 2024

avg_temp_2000 <- avg_temp_map %>% filter(season_year == 2000)
avg_temp_2024 <- avg_temp_map %>% filter(season_year == 2024)

# Create the ggplot for 2000
temp_2000 <- ggplot(avg_temp_2000) +
  geom_polygon(data = us_map, aes(x = long, y = lat, group = group),
               fill = "gray90", color = "white") +
  geom_point(aes(x = LONGITUDE, y = LATITUDE, color = avg_temp), size = 2) +
  scale_color_viridis_c(option = "plasma", name = "Avg. Temp. (°F)") +
  coord_fixed(1.3) +
  facet_wrap(~season) +
  labs(title = "Seasonal Average Temperatures (2000)",
       x = "",
       y = "")+
  theme_minimal()

In [ ]:
# Create the ggplot for 2024
temp_2024 <- ggplot(avg_temp_2024) +
  geom_polygon(data = us_map, aes(x = long, y = lat, group = group),
               fill = "gray90", color = "white") +
  geom_point(aes(x = LONGITUDE, y = LATITUDE, color = avg_temp), size = 2) +
  scale_color_viridis_c(option = "plasma", name = "Avg. Temp. (°F)") +
  coord_fixed(1.3) +
  facet_wrap(~season) +
  labs(title = "Seasonal Average Temperatures (2024)",
       x = "", y = "") +
  theme_minimal()

In [ ]:
# Arrange them to be side by side!

plot_grid(temp_2000, temp_2024, labels=c("", ""), ncol = 1, nrow = 2, widths = c(1), align = "v")

Warning in as_grob.default(plot): Cannot convert object of class numeric into a
grob.

Placing graphs unaligned.

1.  Precipitation Difference between 2000 - 2024 based on seasonality

The precipitation average that is experienced between 2000-2024 for all nine weather stations have no stark difference in patterns across all four seasons (Fig. 3), although it is slightly wetter across the western coast weather stations.

In [ ]:
#PRECIPITATION!!!

#joining the two to relay to the map
avg_prec_map <- avg_prec %>%
  left_join(station_coords, by = "STATION")

#Creating map of avg precipitation for Selected stations based on seasonality!

ggplot(avg_prec_map %>% filter(season_year >= 2000 & season_year <= 2024)) +
  geom_polygon(data = us_map, aes(x = long, y = lat, group = group),
               fill = "gray90", color = "white") +
  geom_point(data = avg_prec_map %>%
               filter(season_year >= 2000 & season_year <= 2024),
             aes(x = LONGITUDE, y = LATITUDE, color = avg_prec), size = 2) +
  scale_color_viridis_c(option = "plasma", name = "Avg. Prec. (100 μm)") +
  coord_fixed(1.3) +
  facet_wrap(~season) +
  labs(title = "Seasonal Average Precipitation by Station (2000–2024)",
       x = "", y = "") +
  theme_minimal()

Precipitation in the all four seasons, Winter, Spring, Summer, and Fall, has dramatically increased from 2000 to 2024, in the nine respective weather stations (Fig. 4).

In [ ]:
# Let's compare first just what 2000 vs 2024 looks like!

# separating 2000 and 2024

avg_prec_2000 <- avg_prec_map %>% filter(season_year == 2000)
avg_prec_2024 <- avg_prec_map %>% filter(season_year == 2024)

# Create the ggplot for 2000
prec_2000 <- ggplot(avg_prec_2000) +
  geom_polygon(data = us_map, aes(x = long, y = lat, group = group),
               fill = "gray90", color = "white") +
  geom_point(aes(x = LONGITUDE, y = LATITUDE, color = avg_prec), size = 2) +
  scale_color_viridis_c(option = "plasma", name = "Avg. Prec. (100 μm)") +
  coord_fixed(1.3) +
  facet_wrap(~season) +
  labs(title = "Seasonal Average Precipitation (2000)",
       x = "", y = "") +
  theme_minimal()

In [ ]:
# Create the ggplot for 2024
prec_2024 <- ggplot(avg_prec_2024) +
  geom_polygon(data = us_map, aes(x = long, y = lat, group = group),
               fill = "gray90", color = "white") +
  geom_point(aes(x = LONGITUDE, y = LATITUDE, color = avg_prec), size = 2) +
  scale_color_viridis_c(option = "plasma", name = "Avg. Prec. (100 μm)") +
  coord_fixed(1.3) +
  facet_wrap(~season) +
  labs(title = "Seasonal Average Precipitation (2024)",
       x = "", y = "") +
  theme_minimal()

In [ ]:
# Arrange them to be side by side!

plot_grid(prec_2000, prec_2024, labels=c("", ""), ncol = 1, nrow = 2, widths = c(1), align = "v")

Warning in as_grob.default(plot): Cannot convert object of class numeric into a
grob.

Placing graphs unaligned.

1.  Bird migration pattern time!

Based on eBird data, we are focusing in the Western Pacific region of the U.S. (Arizona, California, Oregon, and Washington). Based on the eBird data set variables for understanding migratory codes:

In [ ]:
# read in bird data:

amebit_data <- read_csv("data/AB Data/amebit_data.csv")

New names:
• `` -> `...22`
• `` -> `...23`
• `` -> `...24`
• `` -> `...25`
• `` -> `...26`
• `` -> `...27`
• `` -> `...28`
• `` -> `...29`
• `` -> `...30`
• `` -> `...31`
• `` -> `...32`
• `` -> `...33`
• `` -> `...34`
• `` -> `...35`
• `` -> `...36`
• `` -> `...37`
• `` -> `...38`
• `` -> `...39`
• `` -> `...40`
• `` -> `...41`
• `` -> `...42`
• `` -> `...43`
• `` -> `...44`
• `` -> `...45`
• `` -> `...46`
• `` -> `...47`
• `` -> `...48`
• `` -> `...49`
• `` -> `...50`
• `` -> `...51`
• `` -> `...52`
• `` -> `...53`
• `` -> `...54`
• `` -> `...55`

e.g.:
  dat <- vroom(...)
  problems(dat)

Rows: 70649 Columns: 55
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr  (20): OBSERVATION DATE, OBSERVATION COUNT, BREEDING CODE, BREEDING CATE...
dbl   (4): BCR CODE, LATITUDE, LONGITUDE, NUMBER OBSERVERS
lgl  (30): AGE/SEX, ...22, ...28, ...29, ...30, ...31, ...32, ...33, ...34, ...
time  (1): TIME OBSERVATIONS STARTED

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

In [ ]:
ggplot(amebit_monthly, aes(x = state_abb, y = observations, fill = state_abb)) +
  geom_boxplot() +
  scale_fill_brewer(palette = "Set3",
                    name = "State Name",
                    labels = c("AZ" = "Arizona",
                               "CA" = "California",
                               "OR" = "Oregon",
                               "WA" = "Washington")) +
  labs(title = "Distribution of eBird Bittern Observations by State from 2000 - 2024",
       x = "State",
       y = "Observation Count") +
  theme_minimal()

In [ ]:
# Visualize time:


amebit_monthly %>%
  filter(state_abb %in% c("CA", "AZ", "WA", "OR")) %>%
  ggplot(aes(x = month, y = observations, group = year, color = as.factor(year))) +
  geom_line(alpha = 0.5) +
  facet_wrap(~ state_abb, ncol = 2) +
  labs(
    title = "Seasonal Observation Trends of American Bittern (2000–2024)",
    x = "Month",
    y = "Number of Observations",
    color = "Year"
  ) +
  theme_minimal() +
  theme(
    axis.text.x = element_text(angle = 45, hjust = 1),
    strip.text = element_text(face = "bold")
  )

1.  Looking at seasonal changes over time

In [ ]:
annual_summary <- climate_data %>%
  mutate(
    TAVG_F = (TAVG * 9/5) + 32,
    PRCP_mm = PRCP / 10,
    state_abb = str_extract(NAME, "[A-Z]{2}(?=\\sUS)")
  ) %>%
  filter(state_abb %in% c("AZ", "OR", "WA", "CA")) %>%
  group_by(season_year, season, state_abb) %>%
  summarize(
    avg_temp = mean(TAVG_F, na.rm = TRUE),
    avg_precp = mean(PRCP_mm, na.rm = TRUE),
    .groups = "drop"
  ) %>%
  filter(season_year >= 2000, season_year <= 2024)

# Reshaping to make plotting easier to understand shifts in Temp and Precip!

long_annual <- annual_summary %>%
  pivot_longer(cols = c(avg_temp, avg_precp),
               names_to = "variable", values_to = "value")


# Plotting:

ggplot(long_annual, aes(x = season_year, y = value, color = season)) +
  geom_line(linewidth = 1) +
  facet_grid(variable ~ state_abb, scales = "free_y",
             labeller = labeller(variable = c(
               avg_temp = "Avg Temp (°F)",
               avg_precp = "Avg Precip (mm)"
             ))) +
  theme(panel.spacing = unit(1.2, "lines")) +
  scale_x_continuous(breaks = seq(2000, 2024, by = 4)) +
  theme_minimal() +
  theme(
    axis.text.x = element_text(angle = 45, hjust = 1)
  ) +
  labs(
    title = "Seasonal Climate Variation by State (2000–2024)",
    x = "Year", y = NULL, color = "Season"
  ) +
  scale_color_brewer(palette = "Set1")

1.  Let’s combine them to see if there is a difference!

The correlation graph displays the relationship between average seasonal temperature (°F) and the number of Bittern observations, with separate linear trends plotted for each season. The visualized data reveals distinct seasonal patterns in both temperature and observation count (Fig. 7). Each point representing the yearly average temperature and respective count of American Bitterns.

In [ ]:
# Relationship between climate and bittern evolving together!
amebit_seasonal <- amebit_data %>%
  mutate(
    ob_date = mdy(`OBSERVATION DATE`),
    year = year(ob_date),
    month = month(ob_date),
    season = case_when(
      month %in% c(12, 1, 2) ~ "Winter",
      month %in% c(3, 4, 5) ~ "Spring",
      month %in% c(6, 7, 8) ~ "Summer",
      month %in% c(9, 10, 11) ~ "Fall"
    ),
    season_year = if_else(month == 12, year + 1, year),
    state_abb = state.abb[match(STATE, state.name)]
  ) %>%
  filter(state_abb %in% c("CA", "AZ", "WA", "OR")) %>%
  group_by(season_year, season, state_abb) %>%
  summarize(observations = n(), .groups = "drop") %>%
  filter(season_year >= 2000, season_year <= 2024)

# Combining all data sets made so far:
combined_data <- left_join(amebit_seasonal, annual_summary,
                           by = c("season_year", "season", "state_abb"))

combined_long <- combined_data %>%
  pivot_longer(cols = c(avg_temp, avg_precp),
               names_to = "variable", values_to = "value")

In [ ]:
ggplot(combined_long %>% filter(variable == "avg_temp"), 
       aes(x = value, y = observations, color = season)) +
  geom_point(alpha = 0.7) +
  geom_text(data = combined_long %>%
              filter(variable == "avg_temp", season_year %in% c(2000, 2024)),
            aes(label = season_year),
            vjust = -0.5, size = 3, show.legend = FALSE) +
  facet_wrap(~ state_abb, scales = "free_x") +
  labs(
    title = "Seasonal Average Temperature and American Bittern Observations",
    x = "Average Temperature (°F)",
    y = "Number of Observations",
    color = "Season"
  ) +
  theme_minimal() +
  theme(
    axis.text.x = element_text(angle = 0, hjust = 1),
    strip.text = element_text(size = 12),
    legend.position = "bottom"
  ) +
  scale_color_brewer(palette = "Set1")

The correlation graph displays the relationship between average seasonal precipitation (mm) and the number of Bittern observations, with separate linear trends plotted for each season. The visualized data reveals distinct seasonal patterns in both precipitation and observation count (Fig. 8). Each point representing the yearly average precipitation and respective count of American Bitterns.

In [ ]:
# Visualize precipitation seasonality and bittern data:

ggplot(combined_long %>% filter(variable == "avg_precp"), 
       aes(x = value, y = observations, color = season)) +
  geom_point(alpha = 0.7) +
  geom_text(data = combined_long %>%
              filter(variable == "avg_precp", season_year %in% c(2000, 2024)),
            aes(label = season_year),
            vjust = -0.5, size = 3, show.legend = FALSE) +
  facet_wrap(~ state_abb, scales = "free_x") +
  labs(
    title = "Seasonal Average Precipitation and American Bittern Observations",
    x = "Average Precipitation (mm)",
    y = "Number of Observations",
    color = "Season"
  ) +
  theme_minimal() +
  theme(
    axis.text.x = element_text(angle = 0, hjust = 1),
    strip.text = element_text(size = 12),
    legend.position = "bottom"
  ) +
  scale_color_brewer(palette = "Set1")

During Winter, average temperatures ranged from approximately 37°F to 47°F, with Bittern observations generally below 1,000 (Fig. 7). A slight positive trend was noted, suggesting a weak relationship between increasing temperature and higher observations (Fig. 9). In Spring, temperatures were around 50°F to 58°F (Fig. 7). Although this season had the highest overall number of Bittern observations—with some counts exceeding 2,000—there was a slight negative trend in the linear fit, indicating that the number of Bittern observations tended to decrease slightly with increasing temperature (Fig. 9). Fall showed a moderate positive trend, with observations increasing alongside temperatures ranging from 52°F to 59°F (Fig. 7). However, overall observations remained lower compared to Spring and Summer (Fig. 7). Summer displayed the strongest positive trend. As temperatures increased from 67°F to 74°F, Bittern observations increased significantly (Fig. 9). This season also showed more consistent and clustered high observations, suggesting a stronger relationship between higher temperatures and increased presence of Bitterns (Fig. 9).

##### **ANOVA Tests:**

ANOVA Meanings: Most of the variation in American Bittern observations is explained by year (Fig. 7, F = 14.3678, p = \<0.001) and state (Fig. 7, F = 59.0684, p = \<0.001), not directly by average temperature (Fig. 8, F = 2.3702, p = 0.1247) or precipitation (Fig. 9, F = 0.1157, p = 0.7340), rejecting our hypothesis and accepting our null hypothesis. Climate variables (avg. temperature and precipitation) and time of the year has no significant interaction, meaning that the relationship between climate variance and bittern observations appears stable over time within Arizona, California, Oregon, and Washington between 2000 and 2024 (Fig. 10, p = 0.984).

In [ ]:
# 1. creating a linear model test
combined_data$season_year <- as.factor(combined_data$season_year)
combined_data$state_abb <- as.factor(combined_data$state_abb)


lm_mod <- lm(observations ~ avg_temp * season_year + avg_precp * season_year + state_abb, data = combined_data)

library(car)

Loading required package: carData


Attaching package: 'car'

The following object is masked from 'package:dplyr':

    recode

The following object is masked from 'package:purrr':

    some

Anova Table (Type II tests)

Response: observations
                       Sum Sq  Df F value Pr(>F)    
avg_temp                45597   1  2.3702 0.1247    
season_year           6633544  24 14.3678 <2e-16 ***
avg_precp                2225   1  0.1157 0.7340    
state_abb             3408960   3 59.0684 <2e-16 ***
avg_temp:season_year   219475  24  0.4754 0.9840    
season_year:avg_precp  371196  24  0.8040 0.7315    
Residuals             5732734 298                   
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Loading required package: Matrix


Attaching package: 'Matrix'

The following objects are masked from 'package:tidyr':

    expand, pack, unpack

Linear mixed model fit by REML ['lmerMod']
Formula: observations ~ avg_temp + avg_precp + (1 | season_year) + (1 |  
    state_abb)
   Data: combined_data

REML criterion at convergence: 4815.1

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.2167 -0.6062 -0.1211  0.4991  4.4588 

Random effects:
 Groups      Name        Variance Std.Dev.
 season_year (Intercept) 17451    132.1   
 state_abb   (Intercept) 14941    122.2   
 Residual                17908    133.8   
Number of obs: 376, groups:  season_year, 25; state_abb, 4

Fixed effects:
            Estimate Std. Error t value
(Intercept)  81.1051    85.1900   0.952
avg_temp      1.4967     0.8374   1.787
avg_precp    -0.7658     2.3499  -0.326

Correlation of Fixed Effects:
          (Intr) avg_tm
avg_temp  -0.611       
avg_precp -0.462  0.635

Mixed and Random Effects Meaning:

The effect of average temperature is weak but possibly meaningful. Bitterns are observed slightly more in warmer years (Fig. 11 & 12), but this is insignificant, indicating the sample size might be too small to make this assessment concrete (Fig. 10, t = 1.79). Precipitation has no notable effect (Fig. 9, t = -0.33). Most of the variation is still explained by year and state-level random effects, reinforcing the ANOVA results (Fig. 13, \[1\] year variance = 17451, \[2\] state variance = 14941). Based on these results, we can interpret that we are more likely to observe bitterns in recent years, due to the eBird application on phones becoming more accessible for those interested in reporting and finding birds in their local area.

In [ ]:
ggplot(combined_data, aes(x = avg_temp, y = observations)) +
  geom_point(alpha = 0.4) +
  geom_smooth(method = "lm", se = TRUE, color = "steelblue1") +
  labs(
    title = "Effect of Average Temperature on Bittern Observations",
    x = "Average Temperature (°F)",
    y = "American Bittern Observations"
  ) +
  theme_minimal()

`geom_smooth()` using formula = 'y ~ x'

In [ ]:
ggplot(combined_data, aes(x = factor(season_year), y = observations)) +
  geom_boxplot(fill = "dodgerblue") +
  labs(title = "Distribution of Observations by Year",
       x = "Year",
       y = "Observations") +
  theme_minimal() +
  theme(axis.text.x = element_text(angle = 35, hjust = 1))

In [ ]:
library(ggeffects)


Attaching package: 'ggeffects'

The following object is masked from 'package:cowplot':

    get_title

In [ ]:
library(sjPlot)


Attaching package: 'sjPlot'

The following objects are masked from 'package:cowplot':

    plot_grid, save_plot

Sorting each group of random effects ('sort.all') is not possible when 'facets = TRUE'.

Sorting each group of random effects ('sort.all') is not possible when 'facets = TRUE'.

[[1]]


[[2]]

### **Discussion:**

These results suggest that seasonal temperature plays a variable role in Bittern observations, likely reflecting differences in migratory behavior, breeding activity, and habitat use throughout the year.

The strong positive trend in Summer may indicate that Bitterns are more active or easier to observe during warmer months, potentially due to breeding activity or greater availability of wetland habitats. Similarly, the positive trend in Fall could reflect pre-migration behavior or favorable foraging conditions as temperatures cool.

The Spring season, despite having the highest individual observation counts, showed a slight negative trend. This may reflect a peak in migration or nesting activity occurring at lower spring temperatures, after which observations decline as temperatures rise.

Winter observations were consistently low, which aligns with the species’ known migratory behavior—many populations leave colder regions during winter months. The weak positive correlation may suggest that in relatively milder winters, some individuals remain present or return earlier.

Overall, this analysis underscores how temperature and seasonality can influence bird detection and presence. Continued research into these “changes in timing at continental scales” could give more insight into how data like this can be applied across species (Guiden 2019). Further investigation into habitat variables, precipitation, and food availability could clarify the mechanisms behind these seasonal patterns. While migratory birds have always been vulnerable to human-related dangers, additional understanding of climate risks is critical to aiding the survival of such species (Smithsonian 2024). Understanding these relationships is especially important in the context of climate change, which may shift seasonal temperature regimes and, in turn, influence Bittern migration and habitat use.

### **References:**

American bittern overview, all about birds, Cornell Lab of Ornithology. Overview, All About Birds, Cornell Lab of Ornithology. (n.d.). https://www.allaboutbirds.org/guide/American_Bittern/overview

Dahl, T. E. (2011). Status and trends of wetlands in the conterminous United States 2004 to 2009. US Department of the Interior, US Fish and Wildlife Service, Fisheries and Habitat Conservation.

eBird. (2025). eBird: An online database of bird distribution and abundance \[web application\]. eBird, Cornell Lab of Ornithology, Ithaca, New York. Available: http://www.ebird.org. (Accessed: April 16, 2025).

La Sorte, F. A., & Thompson III, F. R. (2007). Poleward shifts in winter ranges of North American birds. Ecology, 88(7), 1803-1812.

Steen, V., Powell, A. N. (2012). “Potential Effects of Climate Change on the Distribution of Waterbirds in the Prairie Pothole Region, U.S.A.,” Waterbirds, 35(2), 217-229. https://doi.org/10.1675/063.035.0204.

Status and trends of wetlands in the conterminous United States 2004 to 2009. US Department of the Interior, US Fish and Wildlife Service, Fisheries and Habitat Conservation.